From a545567524e6a742dc929a7d5fb757fba0994130 Mon Sep 17 00:00:00 2001
From: Nick Matantsev <nick.matantsev@gmail.com>
Date: Wed, 25 Aug 2021 06:49:36 +0000
Subject: [PATCH] Update TCON quirks struct to latest format

---
 drivers/gpu/drm/sun4i/sun4i_tcon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/sun4i/sun4i_tcon.c b/drivers/gpu/drm/sun4i/sun4i_tcon.c
index cb7c6a5eb41862..6f4d23bf9028a8 100644
--- a/drivers/gpu/drm/sun4i/sun4i_tcon.c
+++ b/drivers/gpu/drm/sun4i/sun4i_tcon.c
@@ -1479,6 +1479,7 @@ static const struct sun4i_tcon_quirks suniv_f1c100s_quirks = {
 	 */
 	.has_channel_0		= true,
 	/* .has_channel_1	= true, */
+	.dclk_min_div		= 4,
 };
 
 static const struct sun4i_tcon_quirks sun4i_a10_quirks = {
