From d680a664b6d2ca7a92ddf1256d3a66c81e989523 Mon Sep 17 00:00:00 2001
From: Matthew Clewell <matt@mattclewell.com>
Date: Sun, 12 Jun 2022 14:03:24 -0500
Subject: [PATCH] dts:suniv:f1c100s with lcd add touchscreen support

---
 .../suniv-f1c100s-licheepi-nano-with-lcd.dts  | 32 +++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/arch/arm/boot/dts/suniv-f1c100s-licheepi-nano-with-lcd.dts b/arch/arm/boot/dts/suniv-f1c100s-licheepi-nano-with-lcd.dts
index 9da55cb99274..d1432307c7d7 100644
--- a/arch/arm/boot/dts/suniv-f1c100s-licheepi-nano-with-lcd.dts
+++ b/arch/arm/boot/dts/suniv-f1c100s-licheepi-nano-with-lcd.dts
@@ -23,6 +23,19 @@ panel_input: endpoint@0 {
 			};
 		};
 	};
+
+    i2c0: i2c@1C27000 {
+        compatible = "allwinner,sun6i-a31-i2c";
+        reg = <0x01C27000 0x400>;
+        interrupts = <7>;
+        clocks = <&ccu CLK_BUS_I2C0>;
+        resets = <&ccu RST_BUS_I2C0>;
+        pinctrl-names = "default";
+        pinctrl-0 = <&i2c0_pins>;
+        status = "disabled";
+        #address-cells = <1>;
+        #size-cells = <0>;
+    };
 };
 
 &be0 {
@@ -45,3 +58,22 @@ tcon0_out_lcd: endpoint@0 {
 		remote-endpoint = <&panel_input>;
 	};
 };
+
+&pio {
+    i2c0_pins: i2c0 {
+        pins = "PE11", "PE12";
+        function = "i2c0";
+    };
+};
+
+&i2c0 {
+    pinctrl-0 = <&i2c0_pins>;
+    pinctrl-names = "default";
+    status = "okay";
+
+    ns2009: ns2009@48 {
+        compatible = "nsiway,ns2009";
+        reg = <0x48>;
+    };
+};
+
-- 
2.36.1

