From ac54ccf405a347ab9a47800e12a45ba33e1c6362 Mon Sep 17 00:00:00 2001
From: Matthew Clewell <19787629+mclewell@users.noreply.github.com>
Date: Sun, 2 Jan 2022 09:46:06 -0600
Subject: [PATCH] Fix DTB compilation for boysenboard

---
 arch/arm/dts/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index fd47e408f8..44469a9d88 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -450,6 +450,8 @@ dtb-$(CONFIG_STM32F7) += stm32f746-disco.dtb \
 dtb-$(CONFIG_STM32H7) += stm32h743i-disco.dtb \
 	stm32h743i-eval.dtb
 
+dtb-$(CONFIG_MACH_SUNIV) += \
+	suniv-f1c100s-lcd.dtb
 dtb-$(CONFIG_MACH_SUN4I) += \
 	sun4i-a10-a1000.dtb \
 	sun4i-a10-ba10-tvbox.dtb \
-- 
2.34.1

