From 9ef825e2bd4abd5d3ac2516992aa3837fbcde2a3 Mon Sep 17 00:00:00 2001
From: Matthew Clewell <matt@mattclewell.com>
Date: Fri, 10 Jun 2022 18:40:03 -0500
Subject: [PATCH] f1c100s add licheepinano device tree

---
 arch/arm/dts/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index a1b8231073..5bc4730693 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -451,7 +451,7 @@ dtb-$(CONFIG_STM32H7) += stm32h743i-disco.dtb \
 	stm32h743i-eval.dtb
 
 dtb-$(CONFIG_MACH_SUNIV) += \
-	suniv-f1c100s-lcd.dtb
+	suniv-f1c100s-licheepi-nano.dtb 
 dtb-$(CONFIG_MACH_SUN4I) += \
 	sun4i-a10-a1000.dtb \
 	sun4i-a10-ba10-tvbox.dtb \
-- 
2.36.1

